### PR TITLE
Ndt 602

### PR DESCRIPTION
**Pull request by lebeza**
@shashank-naik @wansooseo 
- styled the buttons in the product details page 

image

image

### CODE REVIEW

File: web/themes/custom/digital_toolkit/src/styles/component/page/_product.scss [hunk 1]
```diff
@@ -34,6 +34,28 @@
...
+    display: inline-block;
+    .button {
+      width: auto;
+      background: url(/themes/custom/digital_toolkit/src/images/external-link-icon-white.png) left no-repeat;
+      padding: 12px 15px 11px 50px;
+      display: inline-block;
+      font-size: 12px;
+      margin-right: 15px;
+      margin-bottom: 16px;
+      background-color: #1C5E86;
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/src/styles/component/page/_product.scss]: Update it to `$primary-blue` please.

File: web/themes/custom/digital_toolkit/src/styles/component/page/_product.scss [hunk 2]
```diff
@@ -34,6 +34,28 @@
...
+      background-position: 4%;
+      -webkit-box-shadow: 0 3px 3px #1F496F;
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/src/styles/component/page/_product.scss]: Update it to `$primary-dark-blue` please.

File: web/themes/custom/digital_toolkit/src/styles/component/page/_product.scss [hunk 3]
```diff
@@ -34,6 +34,28 @@
...
+      -moz-box-shadow: 0 3px 3px #1F496F;
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/src/styles/component/page/_product.scss]: Update it to `$primary-dark-blue` please.

File: web/themes/custom/digital_toolkit/src/styles/component/page/_product.scss [hunk 4]
```diff
@@ -34,6 +34,28 @@
...
+      box-shadow: 0 3px 3px #1F496F;
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/src/styles/component/page/_product.scss]: Update it to `$primary-dark-blue` please.

Comment (lebeza) [reply web/themes/custom/digital_toolkit/src/styles/component/page/_product.scss]: @wansooseo  fixed the color variables. please pull.

**Pull request merged**